torch/backends/cudnn/rnn.py

```diff
@@ -35,20 +35,13 @@ def __setstate__(self, state):
         self.inner = None
 
 
-def init_dropout_descriptor(handle, dropout, train, dropout_seed, dropout_state):
+def init_dropout_state(ty, dropout, train, dropout_seed, dropout_state):
     dropout_desc_name = 'desc_' + str(torch.cuda.current_device())
     dropout_p = dropout if train else 0
     if (dropout_desc_name not in dropout_state) or (dropout_state[dropout_desc_name].get() is None):
         dropout_state[dropout_desc_name] = Unserializable(
-            cudnn.DropoutDescriptor(handle, dropout_p, dropout_seed)
+            torch._C._VariableFunctions._cudnn_init_dropout_state(ty, dropout_p, train, dropout_seed)
```

Comment:
fwiw you can probably do torch._cudnn_init_dropout.

Reply:
OK, I'll fix this and a bunch of others in its own commit.